Database/V1__Create_user_related_tables.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE `sdliticadb`.`user` (
```

Comment:
Can we get rid of SQL in our project from scratch? I want to use Flyway to manage db migrations up and I think that we can use its scripts from very beginning, even for creation of very first table. Could you please check this and if it is possible, update this part?

Reply:
Sure! I am sorry. Can you please provide me some sample?